app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -187,4 +206,34 @@ public void imageClicked() {
   public void wikipediaButtonClicked() {
     callback.addImageToWikipedia(contribution);
   }
+
+  /**
+   * Triggers a callback for pause/resume
+   */
+  @OnClick(R.id.pauseResumeButton)
+  public void onPauseResumeButtonClicked() {
```

Comment:
I guess this was pulled from master, and is not really related to leaderboard?
Don't worry too much, it will probably be solved by rebasing, but in general before rebase before sending a pull request :-)

Reply:
I actually tried to update the leaderboard branch with latest commits

Reply:
Ah yes I see, that's good, thanks! :-)